src/parser/Iri.cpp

```diff
@@ -16,3 +17,15 @@ Iri::Iri(const Iri& prefix, const NormalizedString& suffix) {
 
 // __________________________________________
 NormalizedStringView Iri::getContent() const { return iri_; }
+
+// __________________________________________
+Iri Iri::iriref(const std::string& stringWithBrackets) {
+  return Iri{RdfEscaping::normalizeIriWithBrackets(stringWithBrackets)};
+}
+
+// __________________________________________
+Iri Iri::prefixed(const Iri& prefix, const std::string& suffix) {
+  return Iri{prefix, RdfEscaping::normalizeIriWithoutBrackets(suffix)};
+}
```

Comment:
I think technically this might be subtely wrong, but that is due to preexisting bugs in corner cases of the escaping module, which are unrelated to this PR.
Most notably there actually is not much to "escape" for an iriref, but that requires some other changes by me.

Reply:
In which way might this be subtly wrong?